prisma-fmt/src/get_dmmf.rs

```diff
@@ -0,0 +1,36 @@
+use serde::Deserialize;
+
+/// The implementation of the CLI getDmmf() utility and its JSON format.
+pub mod internal {
+    pub use dmmf::{dmmf_json_from_schema as get_dmmf, *};
+}
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+struct GetDmmfParams {
+    prisma_schema: String,
+}
+
+pub(crate) fn get_dmmf(params: &str) -> String {
+    let params: GetDmmfParams = match serde_json::from_str(params) {
+        Ok(params) => params,
+        Err(serde_err) => {
+            panic!("Failed to deserialize GetDmmfParams: {}", serde_err,);
+        }
+    };
+
+    get_dmmf_impl(params)
+}
+
+fn get_dmmf_impl(params: GetDmmfParams) -> String {
+    // Note:
+    // - if the Prisma schema is not valid, this panics
```

Comment:
That'll need some handling on the TS side to avoid a panic interactive dialog to appear in the CLI whenever the schema is not valid, yes, but given that `get_dmmf` doesn't return a `Result`, there isn't anything different we can do in `prisma-engines` to change this. The TS side shouldn't use `getDmmf` for validation anyway.